must-gather/collection-scripts/gather

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# resource list
+resources=()
+
+# performance operator namespace
+resources+=(ns/openshift-performance-addon)
```

Comment:
ns/openshift-performance-addon-operator?

Reply:
uhm, I thought I picked the one we use. Let me doublecheck

Reply:
The [documentation](https://docs.openshift.com/container-platform-ocp/4.4/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#install-operator-cli_cnf-master) recommends _openshift-performance-addon-operator_

We have been using _openshift-performance-addon_

Reply:
Thanks @gsr-shanks for pointing this out.